### PR TITLE
Fetch fix

### PR DESCRIPTION
**Pull request by signedav**
Do not merge - contains lot of unused debug log.

Replaces the `featureAdded` slot with `committedFeaturesAdded` slot.

Possibly not nice to take the `first` feature out of the list.

Seems to work better but still sometimes the features disappear from the canvas.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-4fd5e2f5f967647557dbedb63eaa0bc49709d787-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-4fd5e2f5f967647557dbedb63eaa0bc49709d787-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-4fd5e2f5f967647557dbedb63eaa0bc49709d787-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-4fd5e2f5f967647557dbedb63eaa0bc49709d787-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-14b441aa0292aaa38057eb536c187cabe4b34263-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-14b441aa0292aaa38057eb536c187cabe4b34263-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-14b441aa0292aaa38057eb536c187cabe4b34263-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-14b441aa0292aaa38057eb536c187cabe4b34263-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-af67b9dfff2b2652f5f4993b8585cedf48e42879-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-af67b9dfff2b2652f5f4993b8585cedf48e42879-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-af67b9dfff2b2652f5f4993b8585cedf48e42879-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-af67b9dfff2b2652f5f4993b8585cedf48e42879-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-1ac176cb11e9322c775e25f42a5fe61efb682e19-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-1ac176cb11e9322c775e25f42a5fe61efb682e19-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-1ac176cb11e9322c775e25f42a5fe61efb682e19-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-1ac176cb11e9322c775e25f42a5fe61efb682e19-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-ad41a4beb01a23c935b0e3ac1eeb5adc070fb760-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-ad41a4beb01a23c935b0e3ac1eeb5adc070fb760-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-ad41a4beb01a23c935b0e3ac1eeb5adc070fb760-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-ad41a4beb01a23c935b0e3ac1eeb5adc070fb760-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-a50fc38288c6eb516ded5fa04e084c961ff545d6-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-a50fc38288c6eb516ded5fa04e084c961ff545d6-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-a50fc38288c6eb516ded5fa04e084c961ff545d6-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-a50fc38288c6eb516ded5fa04e084c961ff545d6-x86-5.apk))

File: src/core/featuremodel.cpp [hunk 1]
```diff
@@ -330,21 +338,21 @@ void FeatureModel::create()
     return;
 
   startEditing();
-  connect( mLayer, &QgsVectorLayer::featureAdded, this, &FeatureModel::featureAdded );
+  connect( mLayer, &QgsVectorLayer::committedFeaturesAdded, this, &FeatureModel::committedFeaturesAdded );
```
Comment (signedav) [inline src/core/featuremodel.cpp]: use the return object here for the disconnect later

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-20eb9488a470f31e200e442995282f80b42a3ed3-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-20eb9488a470f31e200e442995282f80b42a3ed3-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-20eb9488a470f31e200e442995282f80b42a3ed3-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-599-20eb9488a470f31e200e442995282f80b42a3ed3-x86-5.apk))

**Comment by m-kuhn** [comment]
Obsolete

**Pull request closed without merging**